torch/fx/experimental/proxy_tensor.py

```diff
@@ -367,100 +422,32 @@ def __init__(self, tracer):
-        # their stuff).  Furthermore, the tensor argument to lift_fresh is guaranteed
-        # to be freshly allocated, so we want lift_fresh to be a no-op (directly
-        # returning the input argument).
-        #
-        # Here is the basic problem: when we trace this sequence of executions
-        # into an FX graph, what happens to this call sequence?  Traditionally,
-        # tensor constants get interned as buffers on the FX GraphModule.  But
-        # this is dangerous.  Consider:
-        #
-        #       x = torch.tensor(1)
-        #       x.add_(2)
-        #
-        # Naively, this traces into:
-        #
-        #       t = self._tensor_constant0  # initialized to torch.tensor(1)
-        #       x = torch.ops.aten.lift_fresh(t)
-        #       x.add_(2)
-        #
-        # If lift_fresh returns t directly, the subsequent add_ call will
-        # modify the tensor constant. Really, the problem is we've violated
-        # the invariant the the argument to lift is fresh.  So what we should
-        # preserve the invariant by replacing lift_fresh with lift_fresh_copy:
-        #
-        #       t = self._tensor_constant0  # initialized to torch.tensor(1)
-        #       x = torch.ops.aten.lift_fresh_copy(t)
-        #       x.add_(2)
-        #
-        # This is what the overload modification does.
-        else:
-            flat_args = pytree.tree_flatten((args, kwargs))[0]
-            handled_types = [torch.Tensor, _ProxyTensor, torch.nn.Parameter]
-
-            # If there are any tensor subclasses, we need to handle those tensor subclasses first
-            # TODO: we could use types to test this
-            if any(isinstance(arg, torch.Tensor) and type(arg) not in handled_types for arg in flat_args):
-                return NotImplemented
-
-            if func_overload is torch.ops.aten.lift_fresh.default:
-                func_overload = torch.ops.aten.lift_fresh_copy.default
-
-            n_args, n_kwargs = pytree.tree_map_only(SymInt, fetch_symint_proxy(self.tracer), (args, kwargs))
-
-            proxy_out = self.tracer.create_proxy('call_function', func_overload, n_args, n_kwargs,
-                                                 name=self.tracer.graph._target_to_str(func.__name__))
-
-            out = func_overload(*args, **kwargs)
-
-            # If this is a lift, the input tensor is guaranteed to be a
-            # constant, so we keep a copy of the original argument along so
-            # we can query it if we're asked to item() it at some later point
-            is_lift = func_overload is torch.ops.aten.lift_fresh_copy.default
-            if is_lift and out.numel() <= CONSTANT_NUMEL_LIMIT:
-                with maybe_disable_fake_tensor_mode():
-                    constant = args[0].clone()
-            else:
-                constant = None
-            track_tensor_tree(out, proxy_out, constant=constant, tracer=self.tracer)
+        out = proxy_call(self, func, args, kwargs)
```

Comment:
Why still have stuff stubbed here?
If you want to reduce the indent, just immediately call into proxy call at the start.

Reply:
i prefer it this way, since we have a bunch of immediate returns.